presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/properties/LogicalPropertiesImpl.java

```diff
@@ -186,21 +186,9 @@ public String toString()
      * (yet perhaps) propagate or add logical properties. For example, a GroupIdNode does
      * not propagate or add logical properties. This is the PlanNode default.
      */
-    public static class DoNotPropagateBuilder
+    public static LogicalPropertiesImpl doNotPropagateProperties(FunctionResolution functionResolution)
```

Comment:
call this emptyProperties() or something. Ideally this would be a statics final variable, but can't do that until the property classes are no longer mutable.